sdk/nodejs/src/eventQueueAS.ts

```diff
@@ -40,7 +40,7 @@ export class EventQueueAS implements EventQueueInterface {
 
         this.flushInterval = setInterval(this.flushEvents.bind(this), this.flushEventsMS)
 
-        getBucketingLib().initEventQueue(environmentKey, JSON.stringify(options))
+        getBucketingLib().initEventQueue(environmentKey, JSON.stringify(options || {}))
```

Comment:
probably just default the options param to an empty object?

Reply:
yea duh, can't do that in AS, so that's where my brain was.